actionpack/lib/action_dispatch/testing/assertions/response.rb

```diff
@@ -71,6 +71,20 @@ def assert_redirected_to(url_options = {}, options = {}, message = nil)
         assert_operator redirect_expected, :===, redirect_is, message
       end
 
+      # Asserts that the given +text+ is present somewhere in the response body.
+      #
+      #     assert_in_body fixture(:name).description
+      def assert_in_body(text)
+        assert_match(/#{text}/, @response.body)
```

Comment:
Maybe we can escape the Regexp? I can see people using something like `assert_in_body("Name (optional)")` which wouldn't match because of the unescaped parenthesis.

Reply:
Good idea 👍